diff --git a/src/bindings/python/wheel/setup.py b/src/bindings/python/wheel/setup.py
index bbc9465..b9cd8fc 100644
--- a/src/bindings/python/wheel/setup.py
+++ b/src/bindings/python/wheel/setup.py
@@ -443,7 +443,7 @@ pkg_name = os.getenv('WHEEL_PACKAGE_NAME', 'openvino')
 ext_modules = find_prebuilt_extensions(get_dir_list(PY_INSTALL_CFG)) if pkg_name == 'openvino' else []
 
 setup(
-    version=os.getenv('WHEEL_VERSION', '0.0.0'),
+    version=os.getenv('CUSTOM_WHEEL_VERSION', '0.0.0'),
     build=os.getenv('WHEEL_BUILD', '000'),
     author_email=os.getenv('WHEEL_AUTHOR_EMAIL', 'openvino_pushbot@intel.com'),
     name=pkg_name,
